### ISSUE TITLE
Ability to show week number

### ISSUE THREAD

**Issue by georgehrke** [mengxiong10/vue2-datepicker#356]
It would be great if the date picker was able to show the week number next to each week.

Something that looks like this: 
![BD1B4D91-A231-4A80-9FA9-75C79C0E1A24](https://user-images.githubusercontent.com/1250540/65588836-7d2a9600-df88-11e9-8318-a385cd74611b.png)

Thank you for considering my enhancement request!

**Comment by mengxiong10** [comment]
In v3.0.0 plan.

**Comment by mengxiong10** [comment]
v3.0.
Select week.
```html
<date-picker type="week" />
```
Or  Just  show week number.

```html
<date-picker type="date" show-week-number />
```

**Comment by Josefer25** [comment]
This not working for me

**Comment by mengxiong10** [comment]
@Josefer25  It's bug in v3.8.0, I'll fix it in v3.8.1
